panther_analysis_tool/backend/public_api_client.py

```diff
@@ -451,7 +451,7 @@ def _build_client(host: str, token: str) -> GraphQLClient:
 
     transport = AIOHTTPTransport(url=graphql_url, headers={_API_TOKEN_HEADER: token})
 
-    return GraphQLClient(transport=transport, fetch_schema_from_transport=True, execute_timeout=30)
+    return GraphQLClient(transport=transport, fetch_schema_from_transport=False, execute_timeout=30)
```

Comment:
correct. The only protected endpoint atm is around inline filters for simple detections. We need to be careful about moving existing public endpoints to protected since that could break things for all outdated PAT clients.